src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -0,0 +1,135 @@
+/*++

```

Comment:
wow I hate that the github diff for this cannot figure out the file was renamed

Reply:
for future reviewers:
```
git diff --word-diff main:src/cascadia/TerminalApp/TerminalSettings.h origin/dev/cazamor/sui/move-term-sets:src/cascadia/TerminalSettingsModel/TerminalSettings.h
```

that'll give you the diff for this file by words. Spoiler: it's just renaming namespaces